django_sirene/importers.py

```diff
@@ -126,172 +139,192 @@ def _prepare_relateds(self, params, row):
         TODO: Create update method to update label of relateds
         """
         # prepare precreate municipality if needed
-        municipality_id = params.get('municipality_id')
+        municipality_id = params.get("municipality_id")
         if municipality_id and municipality_id not in self.db_municipalities_code:
             municipality = Municipality(
-                code=municipality_id,
-                name=row['LIBCOM']
+                code=municipality_id, name=row["libelleCommuneEtablissement"]
             )
             self.relateds_to_create.add(municipality)
             self.db_municipalities_code.add(municipality.code)
 
-        # prepare precreate activities if needed
-        activity_id = params.get('activity_id')
+        # prepare precreate activity if needed
+        # we will get its name in an other way
+        activity_id = params.get("activity_id")
         if activity_id and activity_id not in self.db_activities_code:
-            activity = Activity(
-                code=activity_id,
-                name=row['LIBAPET']
-            )
+            activity = Activity(code=activity_id, name="")
             self.relateds_to_create.add(activity)
-            self.db_activities_code.add(activity.code)
-
-        # prepare precreate legal_status if needed
```

Comment:
Why don't we need to precreate legal_status anymore?

Reply:
The legal status is now found in the other file, so we process it in the other importer (around lines 240/250)